PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/ui/SparkSubmissionContentPanel.java

```diff
@@ -261,7 +264,15 @@ public void propertyChange(PropertyChangeEvent evt) {
                         GridBagConstraints.WEST, GridBagConstraints.HORIZONTAL,
                         new Insets(margin, margin, 0, margin), 0, 0));
 
-        errorMessageLabels[ErrorMessageLabelTag.ClusterName.ordinal()] = new JLabel("Cluster Name Should not be null");
+        boolean isSignedIn = false;
+
+        try {
+            isSignedIn = AuthMethodManager.getInstance().isSignedIn();
+        } catch (IOException ignored) { }
+
+        errorMessageLabels[ErrorMessageLabelTag.ClusterName.ordinal()] = new JLabel( isSignedIn ?
+                        "Cluster Name Should not be null, please choose one for submission" :
+                        "Can't list cluster, please login within Azure Explorer and refresh");
```

Comment:
>Azure Explorer [](start = 65, length = 14)

"View -> Tool Windows -> Azure Explorer"

Reply:
fix it